README.rst

```diff
@@ -141,28 +141,62 @@ version 8.0, ``pip-compile`` offers ``--generate-hashes`` flag:
     # This file is autogenerated by pip-compile
     # To update, run:
     #
-    #    pip-compile --generate-hashes --output-file requirements.txt requirements.in
+    #    pip-compile --generate-hashes requirements.in
     #
-    click==6.7 \
-        --hash=sha256:29f99fc6125fbc931b758dc053b3114e55c77a6e4c6c3a2674a2dc986016381d \
-        --hash=sha256:f15516df478d5a56180fbf80e68f206010e6d160fc39fa508b65e035fd75130b \
+    click==7.0 \
+        --hash=sha256:2335065e6395b9e67ca716de5f7526736bfa6ceead690adf616d925bdc622b13 \
+        --hash=sha256:5b94b49521f6456670fdb30cd82a4eca9412788a93fa6dd6df72c94d5a8ff2d7 \
         # via flask
-    flask==0.12.2 \
-        --hash=sha256:0749df235e3ff61ac108f69ac178c9770caeaccad2509cb762ce1f65570a8856 \
-        --hash=sha256:49f44461237b69ecd901cc7ce66feea0319b9158743dd27a2899962ab214dac1
-    itsdangerous==0.24 \
-        --hash=sha256:cbb3fcf8d3e33df861709ecaf89d9e6629cff0a217bc2848f1b41cd30d360519 \
+    flask==1.1.1 \
+        --hash=sha256:13f9f196f330c7c2c5d7a5cf91af894110ca0215ac051b5844701f2bfd934d52 \
+        --hash=sha256:45eb5a6fd193d6cf7e0cf5d8a5b31f83d5faae0293695626f539a823e93b13f6 \
+        # via -r requirements.in (line 1)
+    itsdangerous==1.1.0 \
+        --hash=sha256:321b033d07f2a4136d3ec762eac9f16a10ccd60f53c0c91af90217ace7ba1f19 \
+        --hash=sha256:b12271b2047cb23eeb98c8b5622e2e5c5e9abd9784a153e9d8ef9cb4dd09d749 \
         # via flask
-    jinja2==2.9.6 \
-        --hash=sha256:2231bace0dfd8d2bf1e5d7e41239c06c9e0ded46e70cc1094a0aa64b0afeb054 \
-        --hash=sha256:ddaa01a212cd6d641401cb01b605f4a4d9f37bfc93043d7f760ec70fb99ff9ff \
+    jinja2==2.11.1 \
+        --hash=sha256:93187ffbc7808079673ef52771baa950426fd664d3aad1d0fa3e95644360e250 \
+        --hash=sha256:b0eaf100007721b5c16c1fc1eecb87409464edc10469ddc9a22a27a99123be49 \
         # via flask
-    markupsafe==1.0 \
-        --hash=sha256:a6be69091dac236ea9c6bc7d012beab42010fa914c459791d627dad4910eb665 \
+    markupsafe==1.1.1 \
```

Comment:
**To discuss:** that's a huge block of hashes and could distract users. Maybe better to use `django` instead of `flask`? It has less dependencies and as a result lower compile output (also with `--generate-hashes`). Could be done in a followup PR. What do you think?

Reply:
I had the same question in my mind about the distraction, so I'm 👍 on changing that over. I can do it in a followup or now, since it doesn't take that long.